lib/amazonka-core/src/Amazonka/Prelude.hs

```diff
@@ -47,6 +47,7 @@ import qualified Data.ByteString.Builder as ByteString.Builder
 import qualified Data.ByteString.Lazy as ByteString.Lazy
 import Data.CaseInsensitive as Export (CI)
 import Data.Coerce as Export (Coercible, coerce)
+import Data.Either as Export (fromRight)
```

Comment:
I think import this only in the file that needs it. Feels weird to have only `fromRight` in the prelude.